### PR TITLE
[piex] Add piex fuzzer

### PR DESCRIPTION
**Pull request by guidovranken**
As requested in https://github.com/google/oss-fuzz/issues/402.
Finds one crash (recursion stack overflow) so far, which I work around in my fuzzer: https://github.com/guidovranken/piex/blob/fuzzers/fuzzers/tiff_parser.cpp#L121
If you have somebody who will fix it, I will remove the workaround.
More to come but you may already merge.

### CODE REVIEW

**Comment by guidovranken** [comment]
These two fuzzer targets are expected to cover a lot of the project's code; you may merge.

**Comment by guidovranken** [comment]
I've just messaged a PIEX developer about this.

**Comment by guidovranken** [comment]
Oh, it bounced :(

**Comment by inferno-chromium** [comment]
@LeonScroggins - are you ok with this ? we should others like @yujieqin, @ebrauer added in the project.yaml as well.

**Comment by LeonScroggins** [comment]
> @LeonScroggins - are you ok with this ?

I'm not familiar with the Dockerfile etc, but I support fuzzing piex :)

(Also, I don't see what the workaround is, but we should definitely fix the bug!)

> we should others like @yujieqin, @ebrauer added in the project.yaml as well.

Hmm, it looks like the OWNERS file in https://github.com/google/piex is out of date. Perhaps we should add @abestephensg?

**Comment by guidovranken** [comment]
@LeonScroggins the workaround is here: https://github.com/guidovranken/piex/blob/fuzzers/fuzzers/tiff_parser.cpp#L86

(the bug may only manifest if you call ```piex::TiffParser::Parse()``` directly instead of indirectly via ```piex::GetPreviewImageData```)

Also if you are satisfied with my fuzzers, please merge [my branch](https://github.com/guidovranken/piex/tree/fuzzers) into your master. Once that is done I'll instruct the OSS-Fuzz Dockerfile to fetch from upstream directly.

**Comment by inferno-chromium** [comment]
@LeonScroggins - any update here, did you get a chance to integrating it. need your help here please.

**Comment by noell** [comment]
@LeonScroggins not sure how these [1] get updated with fixes, /me seeing that you have updated them for security fixes before.  Please ping me when and if there are updates.

[1] https://github.com/google/piex,  https://android.googlesource.com/platform/external/piex

**Comment by LeonScroggins** [comment]
@abestephensg is listed as an OWNER of piex internally. I think that'd be the right person to look this over.

**Comment by Dor1s** [comment]
@abestephensg, would you be interested in getting notified about bugs found in piex?

**Comment by Dor1s** [comment]
Sent an email internally to Abe and other potential owners.

File: projects/piex/project.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+homepage: "https://github.com/google/piex"
+primary_contact: "guidovranken@gmail.com"
```
Comment (Dor1s) [inline projects/piex/project.yaml]: I'll change the contact as soon as this merges.

**Pull request merged**